src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -61,6 +61,12 @@ public class ScreenRecorder : MonoBehaviour
         [Tooltip("Directory to save state recordings in.  This directory will be created if it does not exist.  If not specific, this will default to 'unity_videos' in your user profile path for your operating system.")]
         public string stateRecordingsDirectory = "";
 
+        [Tooltip("Minimize the amount of criteria in bot segment recordings.  This limits the endCriteria in recorded bot segments to only include objects with count deltas and objects that were under click locations.")]
+        public bool minimizeRecordingCriteria = true;
+
+        [Tooltip("RG-INTERNAL/EXPERIMENTAL: Minimize the amount of mouse movement data capture in bot segment recordings.  This currently breaks replay on any game where camera movement is locked to the mouse, like First Person Shooters.")]
+        public bool minimizeRecordingMouseMovements = false;
```

Comment:
Another way to put this might be "useful where the way that a mouse moves from one click event to another doesn't matter". No need to add that unless it resonates a lot with you